### PR TITLE
Adjusted documentation and CLI info

### PR DESCRIPTION
**Pull request by Altbierjupp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,8 +90,7 @@ vim .env
 To use the wallet with the [public test mint](#test-instance), you need to change the appropriate entries in the `.env` file. 
 
 #### Test instance
-*Warning: this instance is just for demonstration only. Currently, only Lightning deposits work but not withdrawals. The server could vanish at any moment so consider any Satoshis you deposit a donation. I will add Lightning withdrawals soon so unless someone comes up with a huge inflation bug, you might be able to claim them back at a later point in time.*
-
+*Warning: this instance is just for demonstration only. The server could vanish at any moment so consider any Satoshis you deposit a donation. You can withdraw you funds with the function [cashu pay.](#pay-a-lightning-invoice)*
```
Comment (callebtc) [inline README.md:93]:
```suggestion
*Warning: this instance is just for demonstration only. The server could vanish at any moment so consider any Satoshis you deposit a donation. You can withdraw you funds with the command [cashu pay](#pay-a-lightning-invoice).*
```

**Comment by callebtc** [comment]
Thanks for the PR! 

The changes for the CLI help aren't quite appropriate because the command descriptions are not supposed to be very verbose and I'd like to keep them as short as possible. If you take them out, I'll happily merge the README fix though. Would appreciate!

**Comment by Altbierjupp** [comment]
Changes in CLI reverted

File: README.md [hunk 2]
```diff
@@ -90,8 +90,7 @@ vim .env
 To use the wallet with the [public test mint](#test-instance), you need to change the appropriate entries in the `.env` file. 
 
 #### Test instance
-*Warning: this instance is just for demonstration only. Currently, only Lightning deposits work but not withdrawals. The server could vanish at any moment so consider any Satoshis you deposit a donation. I will add Lightning withdrawals soon so unless someone comes up with a huge inflation bug, you might be able to claim them back at a later point in time.*
-
+*Warning: this instance is just for demonstration only. The server could vanish at any moment so consider any Satoshis you deposit a donation. You can withdraw you funds with the command [cashu pay](#pay-a-lightning-invoice).*
```
Comment (callebtc) [inline README.md:93]:
```suggestion
*Warning: this instance is just for demonstration only. The server could vanish at any moment so consider any Satoshis you deposit a donation.*
```

File: cashu/wallet/cli.py [hunk 3]
```diff
@@ -284,6 +284,7 @@ async def burn(ctx, token: str, all: bool, force: bool):
     wallet.status()
 
 
+
```
Comment (callebtc) [inline cashu/wallet/cli.py:287]:
```suggestion
```

**Pull request merged**